bazel-mode.el

```diff
@@ -0,0 +1,240 @@
+(defun sort-strings-longest-first (l)
```

Comment:
Please add the usual header comments, see https://www.gnu.org/software/emacs/manual/html_node/elisp/Library-Headers.html.

Reply:
Done. I followed the example from https://github.com/google/coverage/blob/master/coverage-tests.el.